cli/command/manifest/push.go

```diff
@@ -236,10 +242,15 @@ func pushReferences(ctx context.Context, out io.Writer, client registryclient.Re
 	return nil
 }
 
-func mountBlobs(ctx context.Context, client registryclient.RegistryClient, ref reference.Named, blobs []reference.Canonical) error {
+func mountBlobs(ctx context.Context, client registryclient.RegistryClient, ref reference.Named, blobs []manifestBlob) error {
 	for _, blob := range blobs {
-		if err := client.MountBlob(ctx, blob, ref); err != nil {
-			return err
+		err := client.MountBlob(ctx, blob.canonical, ref)
+		switch err.(type) {
+		case nil:
+		case registryclient.ErrBlobCreated:
+			if blob.os != "windows" {
+				return fmt.Errorf("error mounting %s to %s", blob.canonical, ref)
+			}
```

Comment:
I think this is missing the default case, where we return the `err` if it's not of type `registryclient.ErrBlobCreated`